docs/howto/screenshot.md

```diff
@@ -0,0 +1,27 @@
+# Create Screenshot and paste into an issue
+
+## Description
+The instructions below describe how to capture a screenshot of your screen, either your entire screen or a partial screenshot. These can be pasted into GitHub issues, pull requests or any markdown file. 
+
+## Screenshot Instructions (Mac)
+1. **Full screen**: Hit the Command, Shift and 3 keys together to take a screenshot of the entire screen
+
+![image](https://user-images.githubusercontent.com/6722114/181063086-d14b3e28-9d25-4464-b575-ae58b69ef395.png)
+
+1. **Partial screen**: Hit the Shift, Command and 4 keys together to take a screenshot of a portion of the screen. Crosshairs will appear and select the portion you would like to capture.
+
+![image](https://user-images.githubusercontent.com/6722114/181063417-b991c80e-676a-4005-a449-cdc23d728820.png)
+
+![image](https://user-images.githubusercontent.com/6722114/181063353-c96be768-0763-402b-8a0b-811ad41fb808.png)
+
+1. Paste the partial screenshot into comment box in GitHub.
+
+## Screenshot Instructions (PC)
+1. Using snipping tools to capture screenshots is described [here](https://support.microsoft.com/en-us/windows/use-snipping-tool-to-capture-screenshots-00246869-1843-655f-f220-97299b865f6b).
+1. **Full screen**: Shift, Command, 4 
```

Comment:
I'm not sure if I wrote this correctly (full screen, partial screen and video).

Reply:
Windows does not have a command key afaik.. What do the instructions say?

Reply:
@matentzn Windows now has a "windows logo" button on their keyboard and it has similar function to Mac command button.